pallets/contracts/src/lib.rs

```diff
@@ -183,18 +194,31 @@ decl_module! {
             <pallet_contracts::Module<T>>::update_schedule(origin, schedule)
         }
 
+        /// Enable or disable the extrinsic `put_code` in this module.
+        ///
+        /// ## Erros
```

Comment:
```suggestion
        /// ## Errors
```
Let's also add `## Arguments` etc.